.github/workflows/update-configuration.yml

```diff
@@ -11,48 +11,4 @@ jobs:
     permissions: write-all
 
     steps:
-      - uses: actions/checkout@v4
-
-      - name: Setup node
-        uses: actions/setup-node@v4
-        with:
-          node-version: "20.10.0"
-
-      - name: Install deps and run configuration update
-        run: |
-          yarn install --immutable --immutable-cache --check-cache
-          yarn tsc --noCheck --project tsconfig.json
-
-      - name: Update manifest configuration using GitHub Script
-        uses: actions/github-script@v7
-        with:
-          script: |
-            const fs = require('fs');
-            const path = require('path');
-
-            const { pluginSettingsSchema } = require('./src/types/plugin-input');
-
-            const manifestPath = path.resolve("${{ github.workspace }}", './manifest.json');
-            const manifest = JSON.parse(fs.readFileSync(manifestPath, 'utf8'));
-
-            const configuration = JSON.stringify(pluginSettingsSchema);
-
-            manifest["configuration"] = JSON.parse(configuration);
-
-            const updatedManifest = JSON.stringify(manifest, null, 2)
-            console.log('Updated manifest:', updatedManifest);
-            fs.writeFileSync(manifestPath, updatedManifest);
-
-      - name: Commit and Push generated types
```

Comment:
Is this handled elsewhere?

Reply:
https://github.com/ubiquity-os/action-deploy-plugin